src/ocean/core/VersionCheck.d

```diff
@@ -39,11 +39,23 @@ unittest
 
 private bool hasFeaturesFromFunc ( istring libname, ulong major, ulong minor ) ()
 {
-    struct Library
+    // D1 does not support plain nested imports, D2 prohibits aggregate import
+    // exposure, thus two different approaches:
+
+    version (D_Version2)
     {
-        mixin("import " ~ libname ~ ".LibFeatures;");
+        mixin("import Library = " ~ libname ~ ".LibFeatures;");
+        mixin("return is(typeof(Library.has_features_"
```

Comment:
Could this mixin here and the mixing at the line 59 (they are the same) be outside this `version/else`?